ee/tabby-db/migrations/0011_remove-redirect-uri-field-oauth.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE google_oauth_credential ADD COLUMN redirect_uri VARCHAR(256);
```

Comment:
Please use 12, 11 is already used in the new email settings

Reply:
sure - will wait for #1403